lib/src/services/bedrock_service.dart

```diff
@@ -47,9 +84,26 @@ class BedrockService {
     dio.options.receiveTimeout = 100000;
   }
 
-  Future<Response> post(String endpoint, {Map payload = const {}}) async {
-    return await dio.post(endpoint, data: payload);
+  Future<dynamic> post(
+      String url, dynamic body, Map<String, dynamic>? queryParams) async {
```

Comment:
I think `body` and `queryParams` can be optional arguments because on this [line](https://github.com/bedrockio/bedrock-flutter/pull/2/files#diff-99d7401067a9526a5fce462f5e11b239ff5fb489e74edb39f1501c3873dbba9fR19), we're passing in `null,null` just to satisfy the method signature but I imagine other endpoints where we won't need to pass body or params so it would make sense to have these two arguments optional.

Reply:
Good idea will take care of this